creator/studies/mutations.py

```diff
@@ -105,6 +108,16 @@ class StudyInput(graphene.InputObjectType):
             "Study version, often the provided by the data access authority"
         )
     )
+    short_code = graphene.String(
```

Comment:
Should these be required?

Reply:
yes, i forgot to update after we decided they should be

Reply:
the input is used for updating studies too, so we can't